com.unity.multiplayer.mlapi/Tests/Runtime/Transport/SIPTransport.cs

```diff
@@ -0,0 +1,241 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using MLAPI.Editor;
+using MLAPI.Transports;
+using MLAPI.Transports.Tasks;
+using UnityEngine;
+
+/// <summary>
+/// SIPTransport (SIngleProcessTransport)
+/// is a NetworkTransport designed to be used with multiple MLAPI instances in a single process
+/// it's designed for the MLAPI in a way where no networking stack has to be available
+/// it's designed for testing purposes and it's not designed with speed in mind
+/// </summary>
+[DontShowInTransportDropdown]
```

Comment:
probably should be wrapped into `#if UNITY_EDITOR` since it wouldn't exist on non-Editor